code/src/WijmoProvider/Features/ColumnFilter.ts

```diff
@@ -200,6 +198,35 @@ namespace WijmoProvider.Feature {
             return this._filter.filterDefinition;
         }
 
+        public setColumnFilterOptions(
+            columnID: string,
+            options: Array<string>,
+            maxVisibleOptions?: number
+        ): void {
+            const column = this._grid.getColumn(columnID);
+
+            // for now we only want this to work on text or dropdown columns
+            if (
+                column.columnType === OSFramework.Enum.ColumnType.Text ||
+                column.columnType === OSFramework.Enum.ColumnType.Dropdown
+            ) {
+                this._filter.getColumnFilter(
+                    column.provider
+                ).valueFilter.uniqueValues = Array.from(
+                    new Set<string>(options) // we only want unique values
+                );
+
+                if (maxVisibleOptions)
```

Comment:
Check my last comment, should we confirm if the value is > 0 here instead?

Reply:
Even if its 0, we won't do anything, because it doesn't make sense to show a list with no values, does it?

Reply:
Ok, so should we have a better and explicit condition here? Or a comment?

Reply:
I think we should ignore the 0 since is the empty value for an integer in the SS.

Reply:
but we can check if its greater than 0, to make sure we don't config with awkward values

Reply:
0 is considered falsy, but ok.

Reply:
done